jans-cli-tui/cli_tui/plugins/010_auth_server/ssa.py

```
@@ -235,14 +234,20 @@ def save_ssa(self, dialog):
                 operation_id = 'post-register-ssa'
                 cli_args = {'operation_id': operation_id, 'cli_object': self.cli_object, 'data': new_data}
-                self.app.start_progressing(_("Saving ssa..."))
-                result = await get_event_loop().run_in_executor(self.app.executor, self.app.cli_requests, cli_args)
-                self.app.stop_progressing()
-                ssa = result.json()
+                msg = _("Saving ssa...")
+                result = await common_data.app.run_config_api_operation(cli_args, msg)
+
+                try:
+                    ssa = result.json()
+                except (ConnectionError, TimeoutError, ValueError, RequestException):
```

Ruff (0.15.9)

[error] Undefined name `RequestException`

(F821)